admin/includes/header_navigation.php

```diff
@@ -11,14 +11,14 @@
 
 $menuTitles = zen_get_menu_titles();
 ?>
-<nav class="navbar navbar-default" role="navigation">
+<nav class="navbar navbar-default">
   <!-- Brand and toggle get grouped for better mobile display -->
   <div class="navbar-header">
     <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-adm1-collapse">
       <span class="sr-only">Toggle navigation</span>
-      <span class="icon-bar"></span>
-      <span class="icon-bar"></span>
-      <span class="icon-bar"></span>
+      <span class="icon-bar">&nbsp;</span>
+      <span class="icon-bar">&nbsp;</span>
+      <span class="icon-bar">&nbsp;</span>

```

Comment:
Why all the `&nbsp;` in several files of this PR?
Is it because the markup is invalid without them?
Or is it personal preference against having nothing inside the `span` or `i`/`icon` elements?